$NetBSD: patch-aa,v 1.1 2002/03/31 14:25:51 dmcmahill Exp $

LP64 fix.  This const gets used as an int not long.

--- YSM.h.orig	Tue Feb 19 21:14:27 2002
+++ YSM.h
@@ -135,7 +135,7 @@
 /* Definition of YSM Status */
 
 #define YSM_ONLINE      0x0000
-#define YSM_OFFLINE     (-1L)
+#define YSM_OFFLINE     (-1)
 #define YSM_INVISIBLE   0x0100
 #define YSM_NA          0x0005
 #define YSM_DND         0x0013
